osmtm/views/project.py

```diff
@@ -226,6 +230,67 @@ def project_grid_simulate(request):
     return FeatureCollection([Feature(geometry=shape.to_shape(geometry))])
 
 
+@view_config(route_name='project_clone', permission="project_edit")
+def project_clone(request):
+    _ = request.translate
+    user_id = authenticated_userid(request)
+    user = DBSession.query(User).get(user_id)
+
+    try:
+        id = request.params['project']
+        old_project = DBSession.query(Project).get(id)
+        new_project = Project(old_project.name, user=user)
+
+        for att in old_project.__dict__:
+            if (att[0] != '_' and att not in ['area_id', 'author_id',
+                                              'created', 'done', 'id',
+                                              'invalidated', 'last_update',
+                                              'priority_areas', 'status',
+                                              'tasks', 'validated']):
+                setattr(new_project, att, getattr(old_project, att))
```

Comment:
This whole for section feels a bit hacky, but the only other solution I could come up with is supply a list of attributes to **choose** as opposed to _ignore_.